components/contribution-flow/query-parameters.js

```diff
@@ -167,14 +173,6 @@ export const stepsDataToUrlParamsData = (previousUrlParams, stepDetails, stepPro
     delete data.quantity;
   }
 
-  if (isCrypto) {
-    data.cryptoAmount = parseFloat(stepDetails.cryptoAmount) || previousUrlParams.cryptoAmount || 0;
-    data.cryptoCurrency = stepDetails.currency?.value ? stepDetails.currency.value : previousUrlParams.cryptoCurrency;
-    delete data.amount;
-  } else {
-    delete data.cryptoAmount;
-  }
-
```

Comment:
No completely related to this PR, but moved that above to have it with the rest of the `stepDetails` parsing and make it consistent; we didn't need to remove `data.cryptoAmount` if `isCrypto === false` since it was not set in the first place